otter/log/spec.py

```diff
@@ -66,6 +119,47 @@
 }
 
 
+def halve(l):
+    """
+    Split a sequence in half, biased to the left.
+
+    :param list l: The sequence to split
+    :return: a `tuple` containing both halves of the sequence.
+    """
+    half_index = int(math.ceil(len(l) / 2.0))
+    return (l[:half_index], l[half_index:])
+
+
+def split(render, elements, max_len, calculate_len=len):
+    """
+    Render some elements of a list, where the length (as determined by
```

Comment:
How about "Split given elements into sub-lists ensuring that length of each sub-list is less than max_len"?